src/app/parcels/add/[id]/page.tsx

```diff
@@ -0,0 +1,21 @@
+import { Metadata } from "next";
+import React from "react";
+import AddParcelForm from "@/app/parcels/add/AddParcelForm";
+
+type AddParcelParameters = {
```

Comment:
Can you explain why you chose to use `type` instead of `interface` and why we need `params: { id: string }` instead of just `id: string`?

Reply:
So as far as I can tell from my limited research, generally it is seen as better to use type instead of interface where possible, unless there is a good reason to do otherwise.

Reply:
Not actually that sure of the reasons why, but that why I used type instead. Params is needed as you are taking the id out of the url, not passing it in as props.

Reply:
I think there was a similar discussion previously and the decision was to use interfaces unless there's a specific reason to use types

Reply:
Changed to interface!